.spectral.yaml

```diff
@@ -0,0 +1,20 @@
+extends: spectral:oas
+functions: [x-header-parameter]
+rules:
+  wrong-version:
+    description: OpenApi version must be 3.0.1
+    given: $.openapi
+    severity: error
+    then:
+      field: "@value"
+      function: pattern
+      functionOptions:
+        match: "^3.0.1+$"
```

Comment:
I guess it would be fine if it's 3.x.x right. Else if we upgrade to 3.0.2 it will fail...